packages/backend/src/routers/userRouter.ts

```diff
@@ -103,3 +103,20 @@ userRouter.get('/identities', isAuthenticated, async (req, res, next) => {
         results: identities,
     });
 });
+
+userRouter.patch(
```

Comment:
Why is this different to an `update` endpoint?

Reply:
The update endpoint is used to update the user first name, last name and email. I can refactor the code so the update endpoint can be more flexible if you prefer.

Reply:
Feels right because we want the user to update those settings at anytime anyway (once we implement the UI).

Reply:
@owlas I changed the update endpoint to update the tracking values but the complete endpoint also changes the org name. And the service has difference events etc. Not sure if we should use the same endpoint and service.